File: in_go/src/pattern_6.go
package src

import "fmt"

func Pattern_6() {
	row := 5
	column := 9
	symbol := "*"
	toggle := false

	for i := 0; i < row; i++ {
		toggle = true
		for j := 0; j < column; j++ {
			if j >= row-1-i && j <= row-1+i && toggle {
				toggle = false
				fmt.Print(symbol)
			} else {
				toggle = true
				fmt.Print(" ")
			}
		}
		fmt.Println()
	}
}

File: in_typescript/src/Pattern_6.ts
export default function pattern_6() {
  let row = 5;
  let column = 9;

  const symbol = "*";
  let toggle = false;
  for (let i = 0; i < row; i++) {
    toggle = true;
    for (let j = 0; j < column; j++) {
      if (j >= row - 1 - i && j <= row - 1 + i && toggle) {
        toggle = false;
        process.stdout.write(`${symbol}`);
      } else {
        toggle = true;
        process.stdout.write(" ");
      }
    }
    process.stdout.write("\n");
  }
}
